internal/route/node_filter.go

```diff
@@ -183,21 +183,21 @@ func (f *IsAtMaxHeightForGroup) Apply(_ metadata.RequestMetadata, upstreamConfig
 	return false
 }
 
-type SimpleIsStateOrTracePresent struct {
+type IsArchiveNodeMethod struct {
 	logger *zap.Logger
 }
 
-func (f *SimpleIsStateOrTracePresent) Apply(
+func (f *IsArchiveNodeMethod) Apply(
 	requestMetadata metadata.RequestMetadata,
 	upstreamConfig *config.UpstreamConfig,
 ) bool {
-	if requestMetadata.IsStateRequired || requestMetadata.IsTraceMethod {
+	if requestMetadata.IsStateRequired || requestMetadata.IsTraceMethod || requestMetadata.IsLogMethod {
```

Comment:
Might be worth documenting this in comments + in the README.

This makes our gateway not super usable by outsiders, so we should make it clear.